modules/compute/vm-instance/README.md

```diff
@@ -143,16 +143,18 @@ limitations under the License.
 
 | Name | Version |
 |------|---------|
-| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.14.0 |
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.2.0 |
```

Comment:
I think it would be valuable to add a section to the README on the use of `terraform apply -replace-""` to force replacement of a VM whose family or project have not changed. It would probably a sequence like:

```text
terraform state list
# search for module ID and reosurce
terraform apply -replace="address"
```

Reply:
PTAL